src/EnergyPlus/WaterToAirHeatPumpSimple.hh

```diff
@@ -146,12 +155,13 @@ namespace WaterToAirHeatPumpSimple {
               WaterMassFlowRate(0.0), DesignWaterMassFlowRate(0.0), InletWaterTemp(0.0), InletWaterEnthalpy(0.0), OutletWaterTemp(0.0),
```

Comment:
We are getting rid of these default constructors and initializing in the declaration.